src/tools/pub/custom-package-repositories.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Custom Package Repositories
+description: "How Dart's package management tool, pub, works with custom package repositories."
+---
+
+The `pub` tool has support for third-party package repositories. A package
+repository is an HTTP server from which the `pub` client can fetch packages.
+The official package repository is [pub.dev](https://pub.dev), this repository
+is by the `pub` tool by default. A package repository is identified by a
+_hosted-url_ (e.g. `https://pub.example.com/my-account/`). 
+
+Third-party package servers can be useful for hosting private packages. It is
+also common to use [git-dependencies](./dependencies#git-packages) for hosting
```

Comment:
```suggestion
also common to use [git-dependencies](/dependencies#git-packages) for hosting
```

Reply:
I'm guessing that if we want to make links absolute we have start from `/tools/...`